styles.css

```diff
@@ -256,6 +269,7 @@ textarea {
 
   body {
     flex-flow: column nowrap;
+    z-index: 1;
```

Comment:
Why are you adding a `z-index` to the entire `body` element? This seems a little risky. Is there a way to execute what you're trying to do without it?

Reply:
I was making sure the dropdown menu has a higher index than the body its covering, I can try to take it off and see if everything is still working right

Reply:
Everything looks good still, thanks for the tip